### DISCUSSION TITLE
Welcome! Help us with F5/NGINX documentation.

### DISCUSSION THREAD

**Discussion by mjang** [nginx/documentation#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mjang** [comment]
I'll start the discussion. I'm Mike Jang. I've been working with Linux since 1999. I actually enjoy the challenges associated with Security-Enhanced Linux. I've passed the Red Hat Certified Engineer exam three times, and I enjoy working from the command line.

Some of you may recognize the books that I've written on Amazon. As a Technical Writer, my primary authoring tool is `vi`.

**Comment by ADubhlaoich** [comment]
Hello, I'm Alan Dooley! I work with Mike as part of NGINX's DocOps team: we're responsible for all aspects of documentation delivery at NGINX. Someone on our team is responsible for just about anything you could think of if words or websites are involved.

I used to do DevOps work: prior to that I was a SecOps analyst doing pentesting work. My online presence is pretty transparent if you search for my username: I'm not overly active on social media, but you can piece together a lot about me from my website and online accounts.

I'm based in Dublin, Ireland, and used to be heavily involved with Python Ireland. I've technically been involved with organising events longer than I've had a consistent professional career: there's a non-zero chance I end up volunteering with or running any conferences I attend at a later point.

My main hobby nowadays is swing dancing, and I write far too often about it.

**Comment by jputrino** [comment]
Hello, I'm Jodie Putrino! 👋🏻 I manage the NGINX DocOps team. I live in Denver, CO, and enjoy spending my "AFK" time outside, enjoying all of the natural beauty Colorado has to offer. Outside of work, I'm active in the Write the Docs community as both a member and an organizer for the annual conference in Portland, OR.

**Comment by ChisomUma** [comment]
Hi, My name is Chisom Uma. I help with improving NGINX documentation as an open-source contributor!
